main.sh

```diff
@@ -562,8 +574,10 @@ for checksum in ${checksums[@]+"${checksums[@]}"}; do
 done
 
 if [[ -n "${dry_run}" ]]; then
-  info "skipped upload because action is running in dry-run mode"
-  printf "tag: %s ('dry-run' if tag ref is not start with 'refs/tags/')\n" "${tag}"
+  if [[ -z "${dry_run_intended}" ]]; then
```

Comment:
This will only appear in the log so I don't think it is needed to be removed.

Reply:
If I intend to use this action to build my app for given target and won't upload anyway, this message is just a noise